test/src/test/java/org/corfudb/integration/CorfuStoreBrowserEditorIT.java

```diff
@@ -861,6 +870,10 @@ public void addRecordTest() throws Exception {
             TABLE_NAME, newKeyString, newValString, newMetadataString);
         Assert.assertNotNull(addedRecord);
 
+        CorfuDynamicRecord addedRecord2 = browser.addRecord(NAMESPACE,
+                table2Name, newKeyString, newValString, newMetadataString);
```

Comment:
need to pass null metadata here, right?

Reply:
both null and non-null works here since the metadata of the table is taken into account. But for clarity I can change this to be null as well.